site/src/pages/TemplateSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -148,19 +164,18 @@ export const TemplateSettingsForm: FC<TemplateSettingsForm> = ({
 
         <TextField
           {...getFieldHelpers("max_ttl_ms")}
-          helperText={Language.maxTtlHelperText}
           disabled={isSubmitting}
           fullWidth
           inputProps={{ min: 0, step: 1 }}
           label={Language.maxTtlLabel}
           variant="outlined"
-          // Display seconds from ms
-          value={form.values.max_ttl_ms ? form.values.max_ttl_ms / 1000 : ""}
-          // Convert ms to seconds
-          onChange={(event) =>
-            form.setFieldValue("max_ttl_ms", Number(event.currentTarget.value) * 1000)
-          }
+          type="number"
         />
+        {!form.errors.max_ttl_ms && form.values.max_ttl_ms && (
```

Comment:
Took me a second to parse this: if no form errors for `max_ttl_ms` and there's a value for `max_ttl_ms` _then_ render this helper component. Got it 👍🏼

Reply:
I'll add a comment!